doc/langref.html.in

```diff
@@ -11032,7 +11026,7 @@ pub fn main() void {
         Zig functions. Since C macros operate at the level of lexical tokens, not all C macros
         can be translated to Zig. Macros that cannot be translated will be demoted to
         {#syntax#}@compileError{#endsyntax#}. Note that C code which <em>uses</em> macros will be
-        translated without any additional issues (since Zig operates on the pre-processed source
+        translated without any additional issues (since Zig operates on the processed source
```

Comment:
I think `pre-processed` is explicitly referring to having run the C pre-processor.
Is there any other processing you're meaning to include with this generalization?

Reply:
I read the old doc as "Zig operates on the sources before they've run through the pre-processor" ("pre-processed" to me means "before processing"), but I can see how an interpretation of "Zig operates on the result of pre-processing the source" (where "pre-processed" means "run through preprocessor") also works...  I'm trying to make it clearer that Zig operates on the source code after the pre-processor has had its way (though the original preprocessor commands are left in tact).  Again, since its clear I'm not obviously improving things, I'll undo this for now.

Reply:
Ah! Well, I'm pretty sure the intention of "pre-processed" was to mean "after the pre-processor has processed the sources".
(Compare with "pre-configured", "pre-packaged", etc., which also mean "has been done" and not "before it's been done".)
That's also what the rest of the sentence points out: "... pre-processed source with macros expanded" -> macro expansion is the job of C's preprocessor.

(I guess there's still slight room for being even more pedantic, according to my understanding:
1. Zig provides `@cInclude`, which operates on the unprocessed C source file.
2. It internally uses embedded clang code to preprocess the source file.
3. Then it uses this embedded clang code to generate an AST from the preprocessed source file.
4. Then it implements its translate-c feature by inspecting this AST.

The sentence ascribes steps 3 and 4 to "Zig", while actually only step 4 is Zig-specific code. Although to the user of course, all steps 1-4 are included in (and enabled by efforts of) the Zig executable distribution, so the distinction is only relevant for technical understanding in the first place.)